.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
   "cSpell.words": ["import", "State"],
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
This was breaking the go formatter in the vscode, after discussing with @masskoder we decided to remove it.